src/main/java/com/github/bpiatek/bbghbackend/ninetyminutes/domain/NinetyMinutesCommentsExtractor.java

```diff
@@ -21,6 +22,12 @@
 @Service
 class NinetyMinutesCommentsExtractor implements CommentHtmlExtractor {
```

Comment:
Mega mi sie nie podoba ta klasa. 
Masa magicznych wartości głównie ze względu na to, ze HTML'a na tym portalu pisał jakiś głąb :)
W każdym razie metodą prób i błędów wydaje mi się, że działa.